modules/manage/partials/tiered-storage.adoc

```diff
@@ -1277,20 +1277,24 @@ When you create a topic, you can use remote recovery to download the topic data
 
 IMPORTANT: Remote recovery is only safe when no other clusters are writing to the specified bucket or container.
 
-To create a new topic using remote recovery:
+IMPORTANT: `redpanda.remote.read` has to be enabled when using remote recovery.
+
+To create a new topic using remote recovery (the recovered topic can read and write in the cloud):
```

Comment:
```suggestion
IMPORTANT: You must enable `redpanda.remote.read` when using remote recovery.

To create a new topic using remote recovery (the recovered topic can read and write in the cloud):
```

Reply:
There are 2 consecutive **Important** admonitions here. I suggest they be combined into a single admonition with both points.

Reply:
Thanks, updated!